resources/lang/en/beatmappacks.php

```diff
@@ -12,14 +12,10 @@
         'blurb' => [
             'important' => 'READ THIS BEFORE DOWNLOADING',
             'instruction' => [
-                '_' => "Installation: Once a pack has been downloaded, extract the .rar into your osu! Songs directory.
-                    All songs are still .zip'd and/or .osz'd inside the pack, so osu! will need to extract the beatmaps itself the next time you go into Play mode.
-                    :scary extract the zip's/osz's yourself,
-                    or the beatmaps will display incorrectly in osu! and will not function properly.",
-                'scary' => 'Do NOT',
```

Comment:
I don't think this part should change. The process is still the same.

Reply:
I think all the user should know is that they should extract the beatmappack to the osu songs folder. If you don't want users to extract osz files manually, I think the best way to do that is just to not tell them that they can be extracted manually, the way it's currently phrased just overcomplicates things in my opinion.

Reply:
Will wait for more opinions on this

Reply:
agree with Samaoo (not to prevent people from extracting `.osz`s, but because that's irrelevant here)